nsist/spyder.nsi

```diff
@@ -259,7 +263,23 @@ FunctionEnd
 !macroend
 !define TrimQuotes `!insertmacro _TrimQuotes`
 
+
 Function validate_pre_install
+
+  FindWindow $0 "" "${PRODUCT_NAME}"
+  IntCmp $0 0 notRunning
+    MessageBox MB_YESNO|MB_ICONINFORMATION "${PRODUCT_NAME} is running. You will need to close it first to proceed. Do you want to close ${PRODUCT_NAME} now?" \
+                                            /SD IDYES IDYES Confirm IDNO NoClose
+                                            Confirm:
+                                              MessageBox MB_YESNO|MB_ICONINFORMATION "All unsaved files and changes will be lost. All ${PRODUCT_NAME} running processes will stop. Are you sure you are want to close ${PRODUCT_NAME}?" \
+                                                                                      /SD IDYES IDYES CloseSpyder IDNO NoClose
+                                              CloseSpyder:
+                                                exec '"$sysdir\cmd.exe" /c "taskkill /FI "WINDOWTITLE eq Spyder" /F"'
```

Comment:
@jsbautista I think we could use here [`nsExec::Exec`](https://nsis.sourceforge.io/Docs/nsExec/nsExec.txt) to prevent a cmd from appearing

Reply:
Just in case the change as a suggestion:
```suggestion
                                                nsExec::Exec 'TaskKill /FI "WINDOWTITLE eq Spyder" /F'
```